learn-ocaml.el

```diff
@@ -57,3 +57,28 @@
 		 :fetch t)
        :stderr learn-ocaml-log-buffer
        :buffer learn-ocaml-log-buffer ))
+
+(defun learn-ocaml-file-writter-filter (proc string)
+  (write-region string nil learn-ocaml-temp t)
+  )
+
+(cl-defun learn-ocaml-grade-file (&key id token server dont-submit file)
+  "enables the user to upload a file to the server"
+  (interactive)
+  (set-buffer learn-ocaml-log-buffer)
```

Comment:
I believe this line and the following (`delete-region`) are unneeded.

Indeed, it think it may be convenient that the **\*log\*** buffer is never erased, but grows bit by bit in case we want to see debug info a bit later.